backend/engine/plugins/lib/utils.py

```diff
@@ -28,6 +29,14 @@ def setup_logging(name):
     return log
 
 
+def handle_exception(exc_type, exc_value, exc_traceback):
```

Comment:
Do we know what the types of these parameters are?

Reply:
Yup, these are the classes for all 3: `<class 'type'> <class 'Exception'> <class 'traceback'>`